config/brands.ts

```diff
@@ -1,5 +1,8 @@
 const brands = {
   niebanalna: {
+    storage: {
+      prefix: "niebanalna",
```

Comment:
I think we don't need separate config for this. Let's use brand id directly. (convention over configuration)